src/core/cgroup.c

```diff
@@ -389,6 +391,7 @@ static char *format_cgroup_memory_limit_comparison(char *buf, size_t l, Unit *u,
 
 void cgroup_context_dump(Unit *u, FILE* f, const char *prefix) {
         _cleanup_free_ char *disable_controllers_str = NULL, *cpuset_cpus = NULL, *cpuset_mems = NULL, *startup_cpuset_cpus = NULL, *startup_cpuset_mems = NULL;
+        _cleanup_free_ char *shutdown_cpuset_cpus = NULL, *shutdown_cpuset_mems = NULL;
```

Comment:
break the above line, no need to duplicate

Reply:
Done, though unsure what the coding convention is on line splitting & indentation like this. PTAL :)